integration/client/client_unix_test.go

```diff
@@ -33,28 +32,11 @@ const (
 )
 
 var (
-	testImage    string
+	testImage    = "mirror.gcr.io/library/busybox:latest"
```

Comment:
nit: s/var/const/

Reply:
Can we also have `const testImageWWWDataUID = 33` in this file?

Reply:
Lets go ahead and merge this so the CI doesn't keep failing on us.  We can address this in a followup